pkg/connector/twitterfmt/convert.go

```diff
@@ -0,0 +1,110 @@
+package twitterfmt
+
+import (
+	"cmp"
+	"context"
+	"fmt"
+	"html"
+	"math"
+	"slices"
+	"strings"
+
+	"maunium.net/go/mautrix/bridgev2"
+	"maunium.net/go/mautrix/bridgev2/networkid"
+	"maunium.net/go/mautrix/event"
+
+	"go.mau.fi/mautrix-twitter/pkg/twittermeow/data/types"
+)
+
+func Parse(ctx context.Context, portal *bridgev2.Portal, msg *types.MessageData) *event.MessageEventContent {
+	body := strings.Builder{}
+	bodyHtml := strings.Builder{}
```

Comment:
Oh I missed this one, it should be `HTML` https://go.dev/wiki/CodeReviewComments#initialisms